### DISCUSSION TITLE
DarkThemeToggle fillRule Error

### DISCUSSION THREAD

**Discussion by Jeremyjins** [themesberg/flowbite-react#750] [Q&A] [answered]
I'm using Remix framework.

I have tried many ways to remove fillRule error when I use DarkThemeToggle component.

Error message like this.
"Warning: Prop `fillRule` did not match. Server: "null" Client: "evenodd""

This error only occurred the first render and not happen afterward.
I guess this is hydration issue.

Any suggestion?

**Comment by Jeremyjins** [comment]
One more information. When build and deploy, it doesn't happen. Only dev mode has this error.

**Comment by tulup-conner** [comment, marked as answer]
Follow #770 for updates on this bug.
